### DISCUSSION TITLE
[Feature Request] Sort by series finale date

### DISCUSSION THREAD

**Discussion by jpsnyder** [trakt/showly#814] [Ideas / Feature Requests]
I do a lot of subscription churning, where I would subscribe to some streaming services for just a month to watch a TV show I'm interested in. Therefore, I usually like to wait for the entire season of a show I'm tracking has completed before I start watching it. 

Currently I track this by adding the shows I'm waiting for in a custom list that I check on from time to time. However, it would be convenient if there was an option to sort shows by most recently aired season finales for the current season I'm on in that show.

If this is too much of a niche use case, I understand. Just thought it would be a unique feature I haven't seen in any of the trackers I've seen.

Loving the app you made!

**Comment by michaldrabik** [comment]
Yeah, it's might be a tad too specific case to include as a main sorting for example.
I'm thinking however that it could be done more subtely via Calendar section of Progress. In there when local searching you could type "finale" or something (similar to 1st episode being marked Season Premiere) and it would filter calendar to only show finales which would suit your case.

I'll keep that in mind.

**Reply by jpsnyder** [reply]
That makes a lot of sense. A filter like that would be perfect.
